Sync `editing/run/` from WPT upstream

Ahmad-S792 opened the pull request:
#### b2193fad46428c14a5180305a0fcffc3b63ab042
<pre>
Sync `editing/run/` from WPT upstream

<a href="https://bugs.webkit.org/show_bug.cgi?id=275777">https://bugs.webkit.org/show_bug.cgi?id=275777</a>

Reviewed by Tim Nguyen.

This patch sync upstream tests from:

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/4eecf048519881313cf12a5bd3cd0a99b6c57fb3">https://github.com/web-platform-tests/wpt/commit/4eecf048519881313cf12a5bd3cd0a99b6c57fb3</a>

* LayoutTests/imported/w3c/web-platform-tests/editing/run/formatblock.html:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/formatblock_4001-last-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/editing/run/forwarddelete.html:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/forwarddelete_6001-last-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertorderedlist-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertorderedlist.html:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertparagraph.html:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertparagraph_6001-last-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertunorderedlist-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertunorderedlist.html:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/undo-redo.html:
* LayoutTests/tests-options.json:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/formatblock_4001-5000-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/formatblock_5001-last-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/forwarddelete_6001-7000-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/forwarddelete_7001-last-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertorderedlist_1-1000-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertorderedlist_1001-last-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertparagraph_6001-7000-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertparagraph_7001-last-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertunorderedlist_1-1000-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertunorderedlist_1001-2000-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertunorderedlist_2001-last-expected.txt:

Canonical link: <a href="https://commits.webkit.org/280299@main">https://commits.webkit.org/280299@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/13ce171d3225ad27af10f089ff38982c29be6dfc

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/56173 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/35499 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/8645 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/59779 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/6609 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/43121 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/6803 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/45358 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/4585 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/58202 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/33389 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/48456 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/26349 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/30169 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/5786 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/5613 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/52154 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/6062 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/61462 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/81 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/6183 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/52720 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/81 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/48523 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/52491 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/12455 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/70 "Passed tests") | | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/31326 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/32412 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/33495 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/32159 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/8ffd6f6327521a7555b093faaeb7e27e84e56414. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/30083

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/13ce171d3225ad27af10f089ff38982c29be6dfc. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/30083

Commit:
Sync `editing/run/` from WPT upstream

https://bugs.webkit.org/show_bug.cgi?id=275777

Reviewed by Tim Nguyen.

This patch sync upstream tests from:

Upstream commit: https://github.com/web-platform-tests/wpt/commit/4eecf048519881313cf12a5bd3cd0a99b6c57fb3

* LayoutTests/imported/w3c/web-platform-tests/editing/run/formatblock.html:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/formatblock_4001-last-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/editing/run/forwarddelete.html:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/forwarddelete_6001-last-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertorderedlist-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertorderedlist.html:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertparagraph.html:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertparagraph_6001-last-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertunorderedlist-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertunorderedlist.html:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/undo-redo.html:
* LayoutTests/tests-options.json:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/formatblock_4001-5000-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/formatblock_5001-last-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/forwarddelete_6001-7000-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/forwarddelete_7001-last-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertorderedlist_1-1000-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertorderedlist_1001-last-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertparagraph_6001-7000-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertparagraph_7001-last-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertunorderedlist_1-1000-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertunorderedlist_1001-2000-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/editing/run/insertunorderedlist_2001-last-expected.txt:

Canonical link: https://commits.webkit.org/280299@main

webkit-commit-queue commented:
Committed 280299@main (b2193fad4642): <https://commits.webkit.org/280299@main>

Reviewed commits have been landed. Closing PR #30083 and removing active labels.
